src/constants/country.js

```diff
@@ -262,3 +262,8 @@ export const LANG_TO_DEFAULT_COUNTRY = {
     [LANG.TR]: COUNTRY.TR,
     [LANG.ZH]: COUNTRY.CN
 };
+
+export const INSTALLMENT_COUNTRY = {
```

Comment:
Would it be more appropriate to make this an array?

Reply:
Also, thinking about it -- can we call this `ALLOWED_INSTALLMENT_COUNTRIES` just to be a little more explicit? Otherwise it's not clear whether installments are enabled by default for these countries or just allowed.

Reply:
👍